app/views/categories/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<% if @categories.empty?%>
+    <h1>Hello, currently you don't have any categories to choose from. You can create a new Category with by clicking
+    the button bellow.
+    </h1>
+<%else%>
+<div class="parent-cat">
+ <% @categories.each do |category|%>
+   <a href="<%= category_payments_path(category.id) %>" class="children-cat">       
+            <svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 576 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --><path d="M547.6 103.8L490.3 13.1C485.2 5 476.1 0 466.4 0H109.6C99.9 0 90.8 5 85.7 13.1L28.3 103.8c-29.6 46.8-3.4 111.9 51.9 119.4c4 .5 8.1 .8 12.1 .8c26.1 0 49.3-11.4 65.2-29c15.9 17.6 39.1 29 65.2 29c26.1 0 49.3-11.4 65.2-29c15.9 17.6 39.1 29 65.2 29c26.2 0 49.3-11.4 65.2-29c16 17.6 39.1 29 65.2 29c4.1 0 8.1-.3 12.1-.8c55.5-7.4 81.8-72.5 52.1-119.4zM499.7 254.9l-.1 0c-5.3 .7-10.7 1.1-16.2 1.1c-12.4 0-24.3-1.9-35.4-5.3V384H128V250.6c-11.2 3.5-23.2 5.4-35.6 5.4c-5.5 0-11-.4-16.3-1.1l-.1 0c-4.1-.6-8.1-1.3-12-2.3V384v64c0 35.3 28.7 64 64 64H448c35.3 0 64-28.7 64-64V384 252.6c-4 1-8 1.8-12.3 2.3z"/></svg>
+            <p><%= category.name%></p>
+            <strong><p>$<%= category.total_transaction_amount%></p></strong>
+    <a/>
+</div>
+<%end%>
+<%end%>
+<div class="cat-btn">
+    <%= link_to "Create a New Category", new_category_path, class: "btn bg-primary", style:"color: white"%>
+</div>
+
```

Comment:
- [ ] The NEW TRANSACTION buttom button button that brings a user to to create a new transaction should be at the buttom of the page. This is not the case at the moment. Make sure you style the button as per the given design.
- [ ] The back button is absent on this page. Add this button which when a user clicks, it should lead to the homepage.
- [ ] At the top of the page, the total amount of all the transactions for the said category should display. This is currently missing
- See screenshot below
![image](https://github.com/IsmailChangezi/snap-scan/assets/62849872/41cf8610-1e97-49ba-8305-c2084720daeb)

Reply:
* As mentioned by the previous CR this issue still persists, Kindly style the new transaction button as per the given design. 
![image](https://github.com/IsmailChangezi/snap-scan/assets/52497246/499c525a-9164-4ccf-ae22-52be3f603fa5)